abel/basex.py

```diff
@@ -174,23 +180,23 @@ def basex_core_transform(rawdata, M_vert, M_horz, Mc_vert,
     return IM
 
 
-def _get_left_right_matrices(M_vert, M_horz, Mc_vert, Mc_horz):
+def _get_left_right_matrices(M_vert, M_horz, Mc_vert, Mc_horz, reg):
     """ An internal helper function for no-up/down-asymmetry BASEX:
         given basis sets  M_vert, M_horz, Mc_vert, Mc_horz,
         return M_left and M_right matrices
     """
 
     nbf_vert, nbf_horz = np.shape(M_vert)[1], np.shape(M_horz)[1]
-    q_vert, q_horz = 0, 0  # No Tikhonov regularization
+    q_vert, q_horz = reg, reg  # Tikhonov regularization parameters
```

Comment:
Should the regularization be applied to both the horizontal or to both the horizontal and vertical (as done here)? I think that perhaps it should only be applied to the horizontal...